### PR TITLE
Support MySQLShowSlaveHostsStatement

### LINKED ISSUE #13137: support parsing SHOW SLAVE HOSTS for mysql
Hi community,

This issue can be a good first try.

## Aim
support parse mysql administration statements sql `SHOW SLAVE HOSTS`

## Basic Qualifications
- java
- maven
- antlr4

## Detail
We have completed the clone rule in DALStatement.g4.
But have not yet implement the visit method in `shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java`
To complete this issue, I think we should complete these steps:
- Create `MySQLShowSlaveHostsStatement` class in this package `org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.dal`
- Add rule in `shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/database/visitor/SQLVisitorRule.java`
- Implement the visit method in `shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java`. 
- Finally, add test for parse clone statement in `shardingsphere-test` module
## Example FYI
- Refer to this merged pr：https://github.com/apache/shardingsphere/pull/12979/files

### PR DESCRIPTION
**Pull request by flycash**
Fixes #13137.

I am not sure if we need this implementation and test case. It's very simple.
```g4
showSlavehost
    : SHOW SLAVE HOST
    ;
```

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowSlaveHostsStatement.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.dal;
+
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.dal.DALStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.MySQLStatement;
+
+/**
+ * MySQL show slave hosts statement.
+ */
+public class MySQLShowSlaveHostsStatement extends AbstractSQLStatement implements DALStatement, MySQLStatement {
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowSlaveHostsStatement.java:27]: Add final for MySQLShowSlaveHostsStatement.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowSlaveHostsStatement.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowSlaveHostsStatement.java:28]: Please remove this blank line.

File: shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/dal/ShowSlaveHostsStatementTestCase.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+import lombok.Getter;
+import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.SQLParserTestCase;
+
+/**
+ * MySQL show slave host statement test case.
+ */
+@Getter
+public final class ShowSlaveHostsStatementTestCase extends SQLParserTestCase { 
+
```
Comment (strongduanmu) [inline shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/dal/ShowSlaveHostsStatementTestCase.java:28]: Please remove this useless blank line.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Comment by flycash** [comment]
Hi, I have resolved all issues, please check it

**Review by strongduanmu — APPROVED** [review]
@flycash Great job, merged!

**Pull request merged**